src/EnergyPlus/AirflowNetwork/src/Elements.cpp

```diff
@@ -2846,7 +2846,7 @@ namespace AirflowNetwork {
         // This subroutine solves airflow for a surface crack component
 
         // Using/Aliasing
-        using DataHVACGlobals::VerySmallMassFlow;
+        using HVAC::VerySmallMassFlow;
```

Comment:
Now that this is so short we probably don't need to have the `using` in there anymore.  For later...

Reply:
I am against `using` statements in general.